client/actions/agenda.js

```diff
@@ -184,13 +184,49 @@ const _addToCurrentAgenda = (plannings) => (
 )
 
 /**
+ * Action factory that ask for creating a planning item from the supplied event,
+ * then calls addEventToCurrentAgenda
+ * @param {array} events - The events needed to create the planning items
+ */
+const askForAddEventToCurrentAgenda = (events) => (
+    (dispatch, getState) => {
+        const currentAgenda = selectors.getCurrentAgenda(getState())
+        if (currentAgenda) {
+            dispatch(showModal({
+                modalType: 'CONFIRMATION',
+                modalProps: {
+                    body: `Do you want to add these ${events.length}
+                        events to the current agenda ?`,
+                    action: () => dispatch(addEventToCurrentAgenda(events)),
+                },
+            }))
+        } else {
+            dispatch(showModal({
+                modalType: 'CONFIRMATION',
+                modalProps: {
+                    body: 'You have to select first an agenda',
```

Comment:
This probably should be "You have select an agenda first" or "You have to first select an agenda"